docs/src/tutorial.md

```diff
@@ -8,7 +8,7 @@ and solution vector because we need to pass those objects to `MakiePlotter`.
 
 ## Plot your results
 
-Currently, `surface`, `mesh`, `arrows` and their mutating analogues with `!` are defined for `MakiePlotter`.
+Currently, [`FerriteVis.solutionplot`](@ref), [`FerriteVis.wireframe`](@ref), [`FerriteVis.surface`](@ref), [`FerriteVis.arrows`](@ref) and their mutating analogues with `!` are defined for `MakiePlotter`.
 Due to the nature of the documentation we need `WGLMakie`, however, you can simply exchange any `WGLMakie` call by `GLMakie`.
```

Comment:
Can we put this into a box to highlight it? I mean this can quickly drown, especially if we start to extend the docs.